src/main.cpp

```diff
@@ -19,7 +19,7 @@ LOG_MODULE_REGISTER(main);
 const char* init_error;
 int init_rc;
 
-#if defined(STM32F1) || defined(STM32F3)
+#if defined(STM32F1) || defined(STM32F3) || defined(STM32F4)
```

Comment:
I believe this is unnecessary on F4, as it has an internal pull-up resistor that should get turned off automatically by the HAL.

Reply:
Done.